web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
+    }
+
+    .modal__content {
+        flex-grow: 1;
+
+        .simplebar-content {
+            box-sizing: border-box;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+        }
+    }
+
+    #add-poll-form {
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+        height: 100%;
+
+        & label {
+            font-weight: bold;
+            margin: 5px 0;
+        }
+
+        & input[type="text"] {
+            border: 1px solid hsl(0deg 0% 0% / 20%);
+            outline: 0;
+            height: 1.8rem;
```

Comment:
It's good practice to document where values like these are coming from.

Reply:
The border is from the compose textarea and the topic input (from `compose.css` which is dedicated to the compose area so I couldn't just add this selector for this rule in that file) and the height I honestly just eyeballed since the default seemed a little too cramped.

Should I mention this in a couple comments here?